crawler/main.js

```diff
@@ -2,9 +2,20 @@ import { default as urlJobPopulator } from './urlJobPopulator.js';
 import { default as urlJobProcessor } from './urlJobProcessor.js';
 import { default as dataStore } from '../dataStore/dataStore.js';
 import { default as kue } from 'kue';
+import { default as cli } from 'cli.args';
```

Comment:
nit: This gets kind of cumbersome and I am not sure why it has been written this way before. Could we shorten them all to just `import whateverName from …`?